src/component/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import styles from "./SideBar.module.css";
+import CottageTwoToneIcon from "@mui/icons-material/CottageTwoTone";
+import ExploreTwoToneIcon from "@mui/icons-material/ExploreTwoTone";
+import SubscriptionsTwoToneIcon from "@mui/icons-material/SubscriptionsTwoTone";
+import WatchLaterTwoToneIcon from "@mui/icons-material/WatchLaterTwoTone";
+import ThumbUpTwoToneIcon from "@mui/icons-material/ThumbUpTwoTone";
+import WorkHistoryTwoToneIcon from "@mui/icons-material/WorkHistoryTwoTone";
+import VpnKeyTwoToneIcon from "@mui/icons-material/VpnKeyTwoTone";
+
+const SideBar = () => {
+  const activeClass = ({ isActive }) => (isActive ? "actives" : "");
+  return (
+    <div>
+      <nav className={` ${styles.sidebar_nav}`}>
+        <div className={`${styles.nav_user_profile}`}>
```

Comment:
```js
<nav className={styles.sidebar_nav}>
        <div className={styles.nav_user_profile}>
```
can't you write these styles like this?

Reply:
i can write but in future when you have to add some extra classes to it then you have to come back to this